### DISCUSSION TITLE
How to hide from description this option

### DISCUSSION THREAD

**Discussion by Qewbitech** [Lorenzo0111/QualityArmory#607] [Q&A] [answered]
![Снимок экрана 2024-12-11 220927](https://github.com/user-attachments/assets/41591a25-00b3-4edf-88e8-bab029d2e7f3)
![2024-12-11_22 06 08](https://github.com/user-attachments/assets/c173cf51-e3ef-4f4c-a95c-3d4032eb88d7)

**Comment by Lorenzo0111** [comment, marked as answer]
Set the variant to 0 in the config
